coretex/_task/worker/artifacts.py

```diff
@@ -75,9 +75,12 @@ def track(taskRun: TaskRun) -> Iterator[FileEventHandler]:
 
         for index, artifactPath in enumerate(eventHandler.artifactPaths):
             logging.getLogger("coretexpylib").debug(f">> [Coretex] Uploading {index + 1}/{len(eventHandler.artifactPaths)} - \"{artifactPath}\"")
-            artifact = taskRun.createArtifact(artifactPath, str(artifactPath.relative_to(root)))
 
-            if artifact is not None:
-                logging.getLogger("coretexpylib").debug(f"\tSuccessfully uploaded artifact")
-            else:
-                logging.getLogger("coretexpylib").debug(f"\tFailed to upload artifact")
+            try:
+                artifact = taskRun.createArtifact(artifactPath, str(artifactPath.relative_to(root)))
+                if artifact is not None:
+                    logging.getLogger("coretexpylib").debug(f"\tSuccessfully uploaded artifact")
+                else:
+                    logging.getLogger("coretexpylib").debug(f"\tFailed to upload artifact")
+            except Exception as e:
+                logging.getLogger("coretexpylib").error(f"Error while creating artifact: {e}")
```

Comment:
Add `\t` as prefix to the string when printing out. Print out the exc_info as well as a debug level log